packages/visualisation/Dockerfile

```diff
@@ -12,8 +12,8 @@ COPY .prettierrc .
 ARG REACT_APP_MAPBOX_ACCESS_TOKEN
 ARG REACT_APP_SIMULATOR_URL
 
-RUN echo $REACT_APP_MAPBOX_ACCESS_TOKEN
```

Comment:
Are these lines (12-16) really necessary? injecting the build-args in skaffold.yaml should be enough?